src/DebugUrlTracker.php

```diff
@@ -23,16 +23,34 @@ class DebugUrlTracker
 {
     private $url;
 
+    /**
+     * @var Symfony1Context
+     */
+    private $context;
+
+    public function __construct(Symfony1Context $context)
+    {
+        $this->context = $context;
+    }
+
     public function onKernelResponse(FilterResponseEvent $event)
     {
-        if ($this->url || ! $event->isMasterRequest() || ! \sfContext::hasInstance()) {
+        if ($this->url || !$event->isMasterRequest() || !$this->context->isInitialized()) {
             return;
         }
+
         $response_headers = $event->getResponse()->headers;
+
+        $isJavascript     = (strpos($this->context->getResponse()->getContentType(), 'javascript') !== false);
+        $isAttachment     = (false !== stripos($response_headers->get('Content-Disposition'), 'attachment;'));
+        $isHtml           = (false === $response_headers->has('Content-Type') && false === strpos($response_headers->get('Content-Type'), 'html'));
+        $isXmlHttpRequest = $event->getRequest()->isXmlHttpRequest();
```

Comment:
For local variable names, please follow the standard that the library currently follows. Don't mix `$response_headers` with `$isJavascript` :)